maps/map_files/LV624/gym/30.alternate.dmm

```diff
@@ -0,0 +1,937 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```

Comment:
Shift the NanoMed or the light, both on top of each other looks weird.

Rest is fine.
![bilde](https://user-images.githubusercontent.com/59782240/175812871-71870399-b03a-4dbb-b11c-003e0a14113f.png)